server/graphql/types/src/types/outbound_return_line.rs

```diff
@@ -0,0 +1,92 @@
+use async_graphql::*;
+use chrono::NaiveDate;
+use repository::{ItemRow, StockLineRow};
+use service::{
+    invoice::outbound_return::generate_outbound_return_lines::OutboundReturnLine, ListResult,
+};
+
+#[derive(SimpleObject)]
+pub struct OutboundReturnLineConnector {
+    total_count: u32,
+    nodes: Vec<OutboundReturnLineNode>,
+}
+
+impl OutboundReturnLineConnector {
+    pub fn from_domain(
+        return_lines: ListResult<OutboundReturnLine>,
+    ) -> OutboundReturnLineConnector {
+        OutboundReturnLineConnector {
+            total_count: return_lines.count,
+            nodes: return_lines
+                .rows
+                .into_iter()
+                .map(OutboundReturnLineNode::from_domain)
+                .collect(),
+        }
+    }
+}
+
+pub struct OutboundReturnLineNode {
+    pub return_line: OutboundReturnLine,
+}
+
+impl OutboundReturnLineNode {
+    pub fn from_domain(return_line: OutboundReturnLine) -> OutboundReturnLineNode {
+        OutboundReturnLineNode { return_line }
+    }
+
+    pub fn item_row(&self) -> &ItemRow {
```

Comment:
I only this pattern with `row`, but does exposing `item_row` and `stock_line_row` fit the pattern ok?

Reply:
I like it, and makes sense, thank you